callback_test.go

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2023 The Ebitengine Authors
 
-//go:build darwin
+//go:build darwin || (linux && (!cgo || arm64 || amd64))
```

Comment:
I prefer alphabetical order, so let's write the arches like this:

```
//go:build darwin || (linux && (!cgo || amd64 || arm64))
```

Update the other comments too.